v2/internal/reconcilers/arm/arm_client_cache.go

```diff
@@ -166,13 +175,7 @@ func (c *ARMClientCache) getCredentialFromAnnotation(ctx context.Context, obj ge
 }
 
 func getSecretNameFromAnnotation(credentialFrom string, resourceNamespace string) types.NamespacedName {
```

Comment:
Now we can totally get rid of this method and just use `types.NamespacedName{Namespace: resourceNamespace, Name: credentialFrom}` instead?

Reply:
Fixed